page-templates/evergreen.php

```diff
@@ -1,4 +1,4 @@
-<?php /* Template Name: Evergreen Page */
```

Comment:
This is needed for WordPress to pick up templates.
https://developer.wordpress.org/themes/template-files-section/page-template-files/

An alternative could be this, 
```
/**
 * Template Name: Evergreen Page
 * The template for displaying evergreen pages.
```

Reply:
Ugh! Didn't know that, I'll change it.